cherrypy/lib/httputil.py

```diff
@@ -66,6 +68,31 @@ def protocol_from_http(protocol_str):
     return int(protocol_str[5]), int(protocol_str[7])
 
 
+def passes_if_range_check(request):
+    """Return true if the request does not have an If-Range header, or it passes the conditions contained in the header.
+    In this case, the Range header should be observed. False if other, with the Range header discarded."""
```

Comment:
The description paragraph needs to be separated from the title part with an empty line.